tests/baselines/reference/user/sift.log

```diff
@@ -0,0 +1,7 @@
+Exit Code: 1
+Standard output:
+node_modules/sift/index.d.ts(80,15): error TS2307: Cannot find module './core'.
```

Comment:
oh, they [fixed a typo](https://github.com/crcn/sift.js/commit/4b771ecce7757ae1275deaecc6879411dfeab85d#diff-b52768974e6bc0faccb7d4b75b162c99) and commented out the entire index.d.ts. I'll see if I can get a PR together to fix it.

Reply:
🤦‍♂ "fix typo" *secretly deletes feature*

Reply:
"fix typo" is my new catch-all commit message.